driverkit/config/b7eb0dd65226a8dc254d228c8d950d07bf3521d2/amazonlinux2_5.4.190-107.353.amzn2.x86_64_1.yaml

```diff
@@ -0,0 +1,6 @@
+kernelversion: 1
+kernelrelease: 5.4.188-104.359.amzn2.x86_64
```

Comment:
@sjanulonoks you need to fix this config for the new kernel version `5.4.190-107.353.amzn2.x86_64`